lib/RequestShortener.js

```diff
@@ -47,7 +46,6 @@ class RequestShortener {
 			request = request.replace(this.parentDirectoryRegExp, "!..");
 		if(!this.buildinsAsModule && this.buildinsRegExp)
 			request = request.replace(this.buildinsRegExp, "!(webpack)");
-		request = request.replace(this.nodeModulesRegExp, "/~/");
```

Comment:
I actually have a terrible local hack to replace this regexp at runtime with `/^\0$/` to prevent it from matching 😅